### PR TITLE
refresh services when certs change

### PR DESCRIPTION
**Pull request by timogoebel**
This commits adds code to refresh services when certs change.
Previously, this has been done in puppet-certs module.

### CODE REVIEW

**Comment by timogoebel** [comment]
... we might want to wait for https://github.com/Katello/puppet-katello/pull/171/files before merging this.

File: manifests/init.pp [hunk 1]
```diff
@@ -171,4 +171,11 @@
   Exec['cpinit'] -> Exec['foreman-rake-db:seed']
 
   User<|title == apache|>{groups +> $user_groups}
+
+  # lint:ignore:spaceship_operator_without_tag
+  Certs::Ssltools::Certutil <| |> ~> Service['qpidd']
```
Comment (ekohl) [inline manifests/init.pp]: Can't we catch this in some other way? I'd think the ```Class['certs::qpid']``` would also take care of this.

**Comment by timogoebel** [comment]
Updated in the hope, that it'll fix the pipeline.

**Comment by ekohl** [comment]
@ehelms I am

**Comment by ehelms** [comment]
Thanks @timogoebel !

**Pull request merged**